packages/app-builder-lib/src/version.ts

```diff
@@ -1 +1 @@
-export const PACKAGE_VERSION = "22.11.7"
+export const PACKAGE_VERSION = "22.11.8"
```

Comment:
What if we automated the manner in which this version is pulled?
```
import { version } from '../../package.json`
export const PACKAGE_VERSION = version
```

Reply:
> Btw, why are updating deps required for releasing?

You asked me to do a release (thanks!), but to apply https://github.com/develar/app-builder/pull/60 we need to update app-builder-lib dependency.

Reply:
Ahhh, I see now. Thanks for the quick reply 🙂